types/react-native/test/index.tsx

```diff
@@ -1087,6 +1088,30 @@ const DarkMode = () => {
     return <Text>Is dark mode enabled? {isDarkMode}</Text>;
 };
 
+// VirtualizedList
+// Test inspired by: https://reactnative.dev/docs/virtualizedlist
+const VirtualizedListTest = () => {
+    const DATA = [1, 2, 3];
+
+    const getItem = (data: number[], index: number) => {
+        return  {
+            title: `Item ${data[index]}`
+        };
+    }
+
+    const getItemCount = (data: number[]) => data.length;
+
+    return (
+        <VirtualizedList
+            data={DATA}
+            initialNumToRender={4}
+            renderItem={({ item }) => <Text>{item.title}</Text>}
```

Comment:
Is `item` here `any`? Can you perhaps add an assertion here to illustrate type generic typing coming through?

Reply:
No, it isn't `any` but get a return type of `getItem`.

[getItem](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ea4002b4198e9bc92e8acffddd4a803565263f98/types/react-native/test/index.tsx#L1096)

Reply:
Ah, actually I meant to use one of the type assertion comments, e.g.:

```tsx
<VirtualizedList
            data={DATA}
            initialNumToRender={4}
            renderItem={({ item }) => {
              type ItemType = typeof item; // ExpectType […]
              return <Text>{item.title}</Text>
            }}
/>
```

But I hear what you’re saying and don’t want to hold up this PR too much. For the next time! 😄